web-portal/test/EventCard.spec.js

```diff
@@ -69,11 +69,19 @@ describe('Card component', () => {
   })
 
   test('omits venue when not set', () => {
-    wrapper.setProps({
-      calendar_event: Object.assign(getEvent(), {
-        venue_id: null
-      })
+    wrapper = shallowMount(Card, {
+      localVue,
+      store,
+      stubs: {
+        NuxtLink: RouterLinkStub
+      },
+      propsData: {
+        calendar_event: Object.assign(getEvent(), {
+          venue_id: null
+        })
+      }
```

Comment:
I seriously have no idea why this had to be changed to make the tests pass. It used to work, the component hasn't changed, and one of the three tests that used `setProps` still works fine.